resources/output_schema.json

```diff
@@ -153,12 +153,16 @@
                                 "date": {
                                     "type": "string",
                                     "format": "date"
+                                },
+                                "sdk": {
+                                    "type": "string"
                                 }
                             },
                             "required": [
                                 "tag",
                                 "name",
-                                "date"
+                                "date",
+                                "sdk"
```

Comment:
Hold on, is `sdk` required? In https://github.com/nrfconnect/vscode-nrf-connect/pull/4020 it is displayed optionally, and that perhaps makes sense.

Reply:
I was not able to figure out the sdk versions of several releases. I'll do it once again though.